src/Containers/ImageCarousel.tsx

```diff
@@ -25,7 +25,9 @@ export interface ImageCarouselProps
 
 export const ImageCarousel: React.FC<ImageCarouselProps> = ({
     imageData,
-    onClick = (): void => {},
+    onClick = (): void => {
+        return undefined;
```

Comment:
Weird

Reply:
Either turn off no empty functions, or this is the correct way to type a void function in typescript.

Reply:
Let's turn off no empty functions then @fpunny